docs/api/graphql/operation.mdx

```diff
@@ -55,12 +55,16 @@ const root = {
 graphql.operation(async (req, res, ctx) => {
   const payload = await graphqlRequest(
     schema,
-    req.body,
+    req.body.query,
     root,
     null,
-    req.variables,
+    req.body.variables,
```

Comment:
GraphQL operation variables are parsed and always present in `req.variables`:

https://github.com/mswjs/msw/blob/d48a2afb892e7abfefb7f099097d644a1ca6dbc9/src/graphql.ts#L165-L170

Referencing them in a `req.body` is okay for POST requests, but if you're communicating with a GraphQL server using a GET request, then `req.body` will be `undefined` and this expression throws. `req.variables` has an isomorphic behavior in this sense.